src/pages/project/project.jsx

```diff
@@ -82,7 +82,11 @@ var Project = React.createClass({
     // FIXME: TODO: This should be done by preventDefaulting the touch event, not via CSS.
     document.body.style.overflowY = 'hidden';
     var mode = this.state.params.mode;
-    var isPlayOnly = (mode === 'play' || mode === 'link');
+
+    var isPlayOnly = mode === 'play';
+    var isLink = mode === 'link';
+    var isPlayOrLink = isPlayOnly || isLink;
```

Comment:
Since we've invented a new variable: is this the right name? Can we name it based on what we're using it for now that it's indicative of multiple aspects?